src/applications/representative-search/components/results/SearchResult.jsx

```diff
@@ -218,6 +218,7 @@ const SearchResult = ({
                   contact={contact}
                   extension={extension}
                   onClick={() => recordContactLinkClick()}
+                  disableAnalytics
```

Comment:
Strangely, I don't see the `disable-analytics` prop on [va-telephone](https://design.va.gov/storybook/?path=/docs/components-va-telephone--default) yet 🤔 so there doesn't seem to be a way to stop the internal event. I think it was assumed there wouldn't ever be a reason to make this a custom event.

But I'll create an issue for the design system team to add that prop. In the mean time, you can probably just make this `disable-analytics` (kabab case) and it'll automatically pick it up once the component is updated.

Reply:
Ticket here: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2621